tsconfig.json

```diff
@@ -1,13 +1,15 @@
 {
   "compilerOptions": {
     "allowJs": true,
+    "checkJs": true,
```

Comment:
I think this might be causing problems with the `all-clients` package since it uses `.mjs` with import attributes. I can get this to build if I remove this line (though I understand why you enabled it). Just an FYI

Reply:
Yeah... let's skip it for now.